client/layout/index.jsx

```diff
@@ -162,6 +168,14 @@ class Layout extends Component {
 				}
 			}
 		}
+
+		if ( this.props.isStateRandomlyCleared ) {
```

Comment:
Not sure this makes sense, but do we need a flag that we displayed this notice already? Just in case we mount this component again in the same session.

Reply:
As soon as the notice is displayed we update this Redux state `isStateRandomlyCleared` to be false. So unless the whole app boots up again this state will remain false even if this component remounts.

Reply:
Same is done when using browser-storage.
https://github.com/Automattic/wp-calypso/blob/e7af7f07ed4a6466d2be0b5e5ccd31dc1b6d4aee/client/components/sympathy-dev-warning/index.tsx#L30